origin-growth/src/resources/invite.js

```diff
@@ -55,6 +55,50 @@ class GrowthInvite {
     })
   }
 
+  /**
+   * Creates a referrer - referee connection in case this referee does not have a referrer yet
+   *
+   * @param {string} code - growth invitation code
+   */
+  static async makeReferralConnection(code, walletAddress) {
+    try {
+      const referralLink = await db.GrowthReferral.findOne({
+        where: {
+          refereeEthAddress: walletAddress
+        }
+      })
+      const referrer = await GrowthInvite._getReferrer(code)
+
+      console.log("Referral link: ", JSON.stringify(referralLink))
```

Comment:
let's use logger.debug

Reply:
uh this line somehow survived... will just delete it